homeassistant/components/reolink/binary_sensor.py

```diff
@@ -104,6 +116,13 @@ async def async_setup_entry(
 
     entities: list[ReolinkBinarySensorEntity] = []
     for channel in reolink_data.host.api.channels:
+        entities.extend(
+            [
+                ReolinkPushBinarySensorEntity(reolink_data, channel, entity_description)
+                for entity_description in BINARY_PUSH_SENSORS
```

Comment:
Why do we need to have two different tuples if the check for adding them is the same?
Using only one tuple will reduce duplicated code